test/runtests.jl

```diff
@@ -55,11 +56,11 @@ for (i, x) in enumerate(TestArrays)
 end
 filename = "$tmp/big.npz"
 npzwrite(filename, old)
-new = npzread(filename)
+read_out = npzread(filename)
```

Comment:
I prefer not to have underscores in my variables. I'd call it `output` or just `out`

Just curious: what was the timing output?